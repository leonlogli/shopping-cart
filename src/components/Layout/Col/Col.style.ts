import styled from 'styled-components'
import { Col } from 'styled-bootstrap-grid'

export const StyledCol = styled(Col)``

export default StyledCol
