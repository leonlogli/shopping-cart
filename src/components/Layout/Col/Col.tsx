import React from 'react'
import { ColProps as BaseColProps } from 'styled-bootstrap-grid'

import StyledCol from './Col.style'

export type ColProps = BaseColProps & {
  children: React.ReactNode
}

const Col = ({ children, ...other }: ColProps) => (
  <StyledCol {...other}>{children}</StyledCol>
)

export { Col }
export default Col
