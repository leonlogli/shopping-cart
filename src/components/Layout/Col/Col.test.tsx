import React from 'react'
import { mount } from 'enzyme'
import Col from './index'

describe('<Col />', () => {
  const wrapper = mount(
    <Col>
      <span>test</span>
    </Col>
  )

  it('should have the correct children', () => {
    expect(wrapper.find('span').length).toEqual(1)
  })

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
