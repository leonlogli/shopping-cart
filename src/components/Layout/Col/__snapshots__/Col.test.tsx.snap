// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Col /> should match a snapshot 1`] = `
<Col>
  <Styled(styled.div)>
    <div
      className="cyFIJw"
      data-name=""
    >
      <span>
        test
      </span>
    </div>
  </Styled(styled.div)>
</Col>
`;
